public/app/core/services/keybindingSrv.ts

```diff
@@ -75,6 +76,12 @@ export class KeybindingSrv {
     this.exit();
   }
 
+  toggleNav() {
+    window.location.href = locationUtil.getUrlForPartial(locationService.getLocation(), {
+      '__feature.topnav': (!config.featureToggles.topnav).toString(),
+    });
```

Comment:
need to trigger full page reload as feature toggles are initialized at frontend app boot